client/controller/index.web.js

```diff
@@ -47,6 +48,8 @@ export const ProviderWrappedLayout = ( {
 	const state = store.getState();
 	const userLoggedIn = isUserLoggedIn( state );
 
+	attachLogmein( userLoggedIn );
+
```

Comment:
This is the source of the duplicated /me/sites hits, there is probably a better place to do this, boot/common maybe?

Reply:
Yes, you want to move this to `client/boot/common`, near the `detectHistoryNavigation.start()` call. Which also adds a global DOM listener. Another global DOM listener is installed in `accessibleFocus()` in the same module. But this one runs before the Redux store is created.

At its current location, in `ProviderWrappedLayout`, the listener would be added again and again on each React rerender.

Reply:
Thanks, will do tomorrow!

Reply:
Dropped this into the middlewares function, not middleware but it seems like the best place for this (there is similar non middleware stuff happening later in the method).